src/test/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/SubmissionComponentSpec.scala

```diff
@@ -158,4 +158,15 @@ class SubmissionComponentSpec extends TestDriverComponentWithFlatSpecAndMatchers
     }
   }
 
+  it should "count workflows by submitter" in withDefaultTestDatabase {
```

Comment:
Additional cases:
* intermediate number like 5
* add 2 Workflows of different terminal statuses, and `listSubmittersWithMoreWorkflowsThan(0, WorkflowStatuses.terminalStatuses)`